.pullapprove.yml

```diff
@@ -3,14 +3,30 @@ approve_regex: '^(Approved|lgtm|LGTM|:shipit:|:star:|:\+1:|:ship:)'
 reject_regex: ^Rejected
 reset_on_push: false
 reviewers:
-  members:
-  - adleong
-  - esbie
-  - gtcampbell
-  - klingerf
-  - olix0r
-  - rmars
-  - siggy
-  - stevej
-  name: default
-  required: 2
\ No newline at end of file
+  -
+    members:
+    - adleong
+    - esbie
+    - gtcampbell
+    - klingerf
+    - olix0r
+    - rmars
+    - siggy
+    - stevej
+    name: default
+    required: 2
+  -
+    members:
+    - adleong
+    - esbie
+    - gtcampbell
+    - klingerf
+    - olix0r
+    - rmars
+    - siggy
+    - stevej
+    name: linkerd-admin
+    required: 1
```

Comment:
Then pullapprove requires approval from /all/ members :(

`A value of -1 means that approval is required from all members.`